lib/ipaddr.rb

```diff
@@ -594,8 +654,8 @@ def _reverse
     end
   end
 
-  def _to_string(addr)
-    case @family
+  def self._to_string(addr, family)
```

Comment:
This is now effectively a public method, so it should be given a better name.  Maybe just `to_string`, `format`, or something.  What do you think?